tasks/src/object-detection/about.md

```diff
@@ -1,21 +1,24 @@
-Object detection is to detect instances of objects in different parts of a given image. Object detection models receive an image as an input and output the images with bounding boxes and labels on the objects detected.
-
 ## Use Cases
 
 ### Autonomous Driving
-Object detection is widely used in computer vision for autonomous driving. Self-driving cars use object detection models to detect pedestrians, bicycles, traffic lights and road signs to decide on which step it should take. 
+
```

Comment:
Were there gaps before? (I don't know if I put them there, we changed these pages a lot 😅) AFAIK we would like to have gaps only before headers and not after.

Reply:
The gaps should not make a difference in rendering I think